vcr/holder/oidc4vci_wallet.go

```diff
@@ -170,6 +184,57 @@ func (h wallet) HandleCredentialOffer(ctx context.Context, offer oidc4vci.Creden
 	return nil
 }
 
+func supportedCredentialFormat(offeredCredential map[string]interface{}) bool {
+	switch offeredCredential["format"] {
```

Comment:
Wouldn't
```
return offeredCredential["format"] == oidc4vci.VerifiableCredentialJSONLDFormat
```
suffice? :-p

Reply:
pff.... to easy. 

I will take it out and add a new PR that adds format specific validation. This solves the other todo I added.